src/piped/core.clj

```diff
@@ -73,6 +73,7 @@
            queue-url
            consumer-fn
            transform-fn
+           backoff-fn
```

Comment:
How about a nacker-opts option that gets passed to a function like I described above?